keras/backend/tensorflow_backend.py

```diff
@@ -453,19 +455,24 @@ def is_keras_tensor(x):
         >>> K.is_keras_tensor(np_var) # A numpy array is not a symbolic tensor.
         ValueError
         >>> k_var = tf.placeholder('float32', shape=(1,1))
-        >>> K.is_keras_tensor(k_var) # A variable indirectly created outside of keras is not a Keras tensor.
+        # A variable indirectly created outside of keras is not a Keras tensor.
+        >>> K.is_keras_tensor(k_var)
         False
         >>> keras_var = K.variable(np_var)
-        >>> K.is_keras_tensor(keras_var)  # A variable created with the keras backend is not a Keras tensor.
+        # A variable created with the keras backend is not a Keras tensor.
```

Comment:
Please use `>>> # my comment message here`. Not using `>>>` suggest that this is the output of a print statement or a function.